flybywiresim/aircraft
# feat(payload): flyPad OS 3.2 - multiple gate boarding when multiple boarding points are opened

Fixes #[issue_no]

## Summary of Changes

* Adds multiple gate boarding support
- [x] Services Page
- [x] Proper GSX Support (?)
- [x] Estimated boarding time displays correctly
- [x] GSX Config to enable dual gate for GSX

Not addressed in this PR: 
 - EFB Payload Front End
 - Door Animations
 - Ground Vehicles (ask asobo)

## Screenshots (if necessary)


![image](https://github.com/flybywiresim/aircraft/assets/15316958/ee2dd5a3-9706-4c80-9218-74eac3fdc576)

![FlightSimulator_sUaEUSNy8K](https://github.com/flybywiresim/aircraft/assets/15316958/bd6b5498-a35f-4f68-8287-c992a750efd2)


https://github.com/flybywiresim/aircraft/assets/15316958/dcae9b01-8750-4605-a880-b0f8b2984ccb

Note: Outdated, Aft Right door does not work for boarding, only catering. 

https://github.com/flybywiresim/aircraft/assets/15316958/eb5fa3d5-c0aa-4ae4-8b7a-ae1e41244c03



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Complete Boarding without any doors/jetway attached, this should work as per the usual boarding process before, then deboard the plane entirely and wait for all sounds to clear (boarding etc.)
2. Go to the services page and open 2 doors (Fwd Left and Aft Left) to enable dual-gate boarding.
3. Go back to the payload page and load passengers (with simbrief or manual entry)
4. The estimated boarding time displayed should be half the duration as previously (less than 10 min) 
5. Start a real or fast time boarding and observe that the estimated duration is accurate, and that the passengers are shown as being filled from both the front and back of the aircraft.
6. Check that deboarding also works in exactly the same way but with the passengers being deboarded from both the front and back of the aircraft.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
